src/collection-items-utils.ts

```diff
@@ -0,0 +1,20 @@
+import { DecentralandAssetIdentifier } from './types'
+
+/**
+ * Checks if the URN follows the standard ERC-721 by having the tokenId identifier at the end
+ * @public
+ */
+export function isExtendedUrn(asset: DecentralandAssetIdentifier): boolean {
+  return ['blockchain-collection-v1-item', 'blockchain-collection-v2-item'].includes(asset.type)
+}
+
+/**
+ * Takes an URN adhering to the ERC-721 standard and resolves both its asset URN and token identity
+ * WARNING: it assumes the URN received follows the ERC-721 standard and that its last part refers to the token id
+ * @public
+ */
+export function getTokenIdAndAssetUrn(completeUrn: string): { assetUrn: string; tokenId: string } {
```

Comment:
shouldn't assetUrn be of type urn (not quite sure how it's called, perhaps DecentralandAsset.. something)

Reply:
I'm not sure about this, I was wondering that may parsing an urn inside this function could be a timing-consuming action and I'd like to keep it simple so I can do this while iterating many URNs in different workflows.